install.py

```diff
@@ -88,9 +88,11 @@ def setup_ramalama(bindir, tmp_dir):
     syspath = "/usr/share/ramalama"
     if sys.platform == "darwin":
         install_mac_dependencies()
-        sharedirs = ["/opt/homebrew/share", "/usr/local/share"]
+        sharedirs = ["/opt/homebrew/share", "/usr/local/share", "/usr/share/ramalama"]
         syspath = next((d for d in sharedirs if os.path.exists(d)), None)
         syspath += "/ramalama"
+    if sys.platform == "darwin":
```

Comment:
You are already in a "darwin" if block?

Reply:
This should be:

```
    sharedirs = ["/opt/homebrew/share", "/usr/local/share", "/usr/share/ramalama"]
    syspath = next((d for d in sharedirs if os.path.exists(d)), None)
    syspath += "/ramalama"
    if sys.platform == "darwin":
        install_mac_dependencies()
```

only one if statement

Reply:
After getting a new macbook... I realise now, modern macOS ships a really old bash binary (the latest GPLv2 version)... But after installing chrome it pulls in /usr/bin/python3 somehow (I think there's some extended macOS tools that can be dragged in like python3)... Maybe we should have left this install script as /bin/bash

Reply:
Excellent, now you can play with ramalama and podman-machine (Desktop).

Reply:
Yup, I remember you brought up in a discussion today podman-machine vs native macOS support... IMO we should do both, unless one starts to get hard to maintain...

native macOS performs better, people notice this and podman-machine is Linux and closer to a server environment...

Reply:
I think this becomes the --nocontainer switch. We add a Environment variable to cause this to be default.

Reply:
We also need to work with Sergio to get better performance.

Reply:
We should but it's gonna be hard to beat macOS... llama.cpp's initial goal was to run well on macOS and Apple Silicon, a ton of projects are using it that way, so that's a lot of combined engineering efforts, Ollama being one tool that uses it this way...

There's also the whole making sure the Linux VM allocates huge amounts of RAM to the VM, etc.

I do think I will port all this back to bash...

It's just the install experience on macOS really sucks via python3...

And bash is probably easier for this kind of thing anyway...